extension/src/popup/components/accountHistory/HistoryItem/index.tsx

```diff
@@ -102,7 +102,7 @@ export const HistoryItem = ({
         <div className="HistoryItem__operation">
           {isPayment ? operationAssetCode : operationString}
           <div className="HistoryItem__date">
-            {recipientLabel} • {date}
+            {isPayment ? `${recipientLabel} \u2022` : null} {date}
```

Comment:
UI fix to only show the `Received/Sent` label if it's a payment. Otherwise, just show the date